docs/user-namespace.md

```diff
@@ -8,11 +8,11 @@ URL: http://madrid.gobierto.dev/user/login
 
 Available users:
 
-| Email               | Password | Notes                                                         |
-| ---                 | ---      | ---                                                           |
-| dennis@gobierto.dev | gobierto | regular user, madrid.gobierto.dev as source site              |
-| reed@gobierto.dev   | gobierto | regular user, unconfirmed, madrid.gobierto.dev as source site |
-| susan@gobierto.dev  | gobierto | regular user, santander.gobierto.dev as source site           |
+| Email               | Password | Notes                                                                        |
+| ---                 | ---      | ---                                                                          |
+| dennis@gobierto.dev | gobierto | regular user, madrid.gobierto.dev as source site                             |
```

Comment:
Good catch!

Dennis isn't actually verified against census at User level (the `User#census_verified` attribute) although there would be a census item that will verify him on next tries if that were seeded as well. Anyways I totally agree on being explicit about those seeds so let's mark Dennis and Reed as not verified against census 👍